corehq/apps/userreports/reports/view.py

```diff
@@ -321,3 +337,26 @@ def excel_response(self):
         file = StringIO()
         export_from_tables(self.export_table, file, Format.XLS_2007)
         return file
+
+
+class CustomConfigurableReportDispatcher(ReportDispatcher):
+    slug = prefix = 'custom_configurable'
+    map_name = 'CUSTOM_UCR'
+
+    def dispatch(self, request, *args, **kwargs):
+        domain = kwargs.get('domain')
+        report_config_id = kwargs.get('report_config_id')
+        class_path = CustomReportConfiguration.report_class_by_domain_and_id(
+            domain, report_config_id
+        )
+        report_class = to_function(class_path)
```

Comment:
My plan is to define the custom UCRs in the source code.

Example: ```custom/abt/reports/views.py```
```
class FormattedSupervisoryReport(ConfigurableReport):
    @property
    def excel_response(self):
...
```

and in the CustomReportConfiguration set ```custom_configurable_report = "custom.abt.reports.views.FormattedSupervisoryReport"```

So nothing is dependent on the database.